Fixed Slovak translation messages.json

dodog opened the pull request:
Fixed Slovak translation

Bilb commented:
Hey,
thanks for the translations.
We use Crowdin to keep the translations up to date. Could you check what needs to be updated there based on your changes? 
https://crowdin.com/project/session-desktop
Thanks

dodog commented:
I translated all remaining strings for Session Desktop, iOS and Android and fixed incorrect translations on Crowdin few months ago, but it does not seems to sync with Github. 
Since then few Session versions has been released, but the Slovak translation is still full of typos, incorrect translations and English strings. 
So I figured that Crowdin is not used anymore.

A few months ago I translated all the remaining strings for Session Desktop, iOS and Android and fixed the incorrect translations on Crowdin, but it doesn't seem to sync with Github. 
Since then, several versions of Session have been released, but the Slovak translation is still full of typos, incorrect translations and English strings. 
So I concluded that Crowdin is no longer used for translations and that is why I created a pull request here.

Bilb commented:
Hey, sorry for the delay I've just pulled the changes from crowdin and your changes seems to be there, in this PR https://github.com/oxen-io/session-desktop/pull/2958/
